Jasmine/Grid/ViewModels/GridViewModel.swift

```diff
@@ -33,6 +33,10 @@ class GridViewModel: GridViewModelProtocol {
         }
     }
 
+    init() {
+        gridData = TextGrid(fromInitialGrid: answers, randomized: true)
```

Comment:
Nope, because `answers` is a property. This is changed in my `finalize-grid-game` branch though.